public/sass/_variables.dark.scss

```diff
@@ -129,7 +132,7 @@ $list-item-shadow: $card-shadow;
 $empty-list-cta-bg: $gray-blue;
 
 // Scrollbars
-$scrollbarBackground: #404357;
+$scrollbarBackground: #aeb5df;
```

Comment:
accidental change?

Reply:
Yes

Reply:
noticed that the scrollbars looked different in the dark theme

Reply:
I have noticed that yestarday as well and forgot to change. My bad.